### ISSUE TITLE
HTML description is covered by chart when chart size is modified

### ISSUE THREAD

**Issue by bvenn** [plotly/Plotly.NET#217]
### Description

When a description is attached to a chart and the size of the chart is modified, it overlaps the description text.

### Repro steps

```
#r "nuget: Plotly.NET, 2.0.0-preview.9"
open Plotly.NET

Chart.Point([1,3])
|> Chart.withSize(500.,635.)
|> Chart.withDescription (ChartDescription.create "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do" "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod")
|> Chart.show
```

### Expected behavior

Description is attached at the bottom of the chart.

### Actual behavior

The chart overlaps with the description.

![image](https://user-images.githubusercontent.com/28917670/135868749-4aae511b-b1bd-444e-83ce-0b95faee998f.png)

**Comment by kMutagene** [comment]
should be resolved @ 2.0.0-preview.14, can you confirm @bvenn ?

**Comment by bvenn** [comment]
Absolutely 🚀
